configsuite/schema.py

```diff
@@ -72,11 +78,19 @@ def assert_valid_schema(schema):
         raise TypeError("Unknown base container: {}".format(schema))
 
 
-def _assert_valid_schema_level(schema):
+def _assert_valid_schema_level(schema, allow_default=True):
     level_schema = copy.deepcopy(schema)
     if MK.Content in level_schema:
         level_schema.pop(MK.Content)
 
+    if MK.Default in schema:
+        if not allow_default:
+            fmt = "Default value is only allowed for contents in NamedDict"
+            raise ValueError(fmt)
```

Comment:
I would really like to add the name of the schema that led to this error, unsure how to do this without passing another argument to the function.. quickly becomes lots of minor arguments here

Reply:
Let us leave it for now. The ease of this validation is not the most importance as it should never hit anyone but the developer. But, when that is said, passing down the key path similarly to in the data validation is definitively possible and useful. Suggest we create an issue for this and implement it broadly in the schema validation if it is to carried out ;)